tsl/src/continuous_aggs/materialize.h

```diff
@@ -40,5 +41,6 @@ void continuous_agg_update_materialization(Hypertable *mat_ht, SchemaAndName par
 										   const NameData *time_column_name,
 										   InternalTimeRange new_materialization_range,
 										   InternalTimeRange invalidation_range, int32 chunk_id);
-
+void mattablecolumninfo_init(MatTableColumnInfo *matcolinfo, List *grouplist);
+void mattablecolumninfo_addinternal(MatTableColumnInfo *matcolinfo);
```

Comment:
Shouldn't both functions be part of `common.h` and `common.c` because is where `MatTableColumnInfo` struct is defined.